### PR TITLE
Adds super special sprites for survival boxes players spawn with.

### PR DESCRIPTION
**Pull request by Probe1**
survival

Plasma man is currently unused.

🆑 

 * sprites:   Survival boxes now have four unique sprites: Vanilla, Vox, Engineer, and ERT.  Spritework by Blithering.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -82,6 +85,8 @@
 
 /obj/item/weapon/storage/box/survival/engineer/New()
 	..()
+	icon_state = "box_eva"
```
Comment (9600bauds) [inline code/game/objects/items/weapons/storage/boxes.dm:88]: whyyy are you setting these in New()

Comment (Probe1) [reply code/game/objects/items/weapons/storage/boxes.dm:88]: Boy I knew when I did this quickly, easily, and without extra work it'd end up being people bitching about me doing what whoever coded it did.

Comment (9600bauds) [reply code/game/objects/items/weapons/storage/boxes.dm:88]: Okay, so you want this to be like...
```
/obj/item/weapon/storage/box/survival/vox
 	icon_state = "box_vox" //you can even set a name or whatever you want here
/obj/item/weapon/storage/box/survival/vox/New()
 	..()
 	for(var/atom/A in src)
 		qdel(A)
 	new /obj/item/clothing/mask/breath/vox(src)
 	new /obj/item/weapon/tank/emergency_nitrogen(src)
 	new /obj/item/stack/medical/bruise_pack/bandaid(src)
```

Comment (Probe1) [reply code/game/objects/items/weapons/storage/boxes.dm:88]: In my head my commit was added before your comment.

**Comment by 9600bauds** [comment]
👌

**Pull request merged**